tests/gocase/unit/type/list/list_test.go

```diff
@@ -84,3 +84,154 @@ func BenchmarkLTRIM(b *testing.B) {
 		})
 	}
 }
+
+func TestZipList(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{
+		"list-max-ziplist-size": "16",
+	})
+	defer srv.Close()
+	ctx := context.Background()
+	rdb := srv.NewClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
+
+	rand.Seed(0)
```

Comment:
Does there need deterministic of PRNG ?

Reply:
The main reason to have seed set here is to make the testing failure reproducible (although this is to predetermine the test input). 
Otherwise, we need other machanisms to ensure failed tests can be reproduced, such as dumping all history commands and replay them...